.vscode/settings.json

```diff
@@ -3,6 +3,6 @@
   "editor.formatOnSave": true,
   "editor.formatOnPaste": false,
   "[python]": {
-    "editor.defaultFormatter": "ms-python.black-formatter"
+    "editor.defaultFormatter": "charliermarsh.ruff"
```

Comment:
I need someone who uses VSCode to test that this is the correct change, and that it works as expected in place of black.